packages/api/src/external/carequality/__tests__/cq-patient-data.ts

```diff
@@ -0,0 +1,107 @@
+import { faker } from "@faker-js/faker";
+import { LinkDemographicsHistory } from "@metriport/core/domain/patient-demographics";
+import dayjs from "dayjs";
+import { ISO_DATE } from "../../../shared/date";
+//import { CQPatientDataModel } from "../models/cq-patient-data";
```

Comment:
nit: left-over

Reply:
just merged 🤦 -- will add to master issue

Reply:
fixed as merged didn't build